### PR TITLE
Add ESM and ESMFold training notebooks

### PR DESCRIPTION
**Pull request by Rocketknight1**

### CODE REVIEW

**Comment by Rocketknight1** [comment]
I've added the token classification section so this notebook should be ready to go!

File: examples/protein_language_modeling.ipynb [hunk 1]
```diff
@@ -0,0 +1,2194 @@
+{
```
Comment (sgugger) [inline examples/protein_language_modeling.ipynb:1]: <p>I think an install from source will be needed for now.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/243/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (sgugger) [inline examples/protein_language_modeling.ipynb:1]: <p>Same here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/243/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (sgugger) [inline examples/protein_language_modeling.ipynb:1]: <p>That section is a bit dry! A couple of comments between each code cell would be nice!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/243/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='83'/>

**Review by sgugger — APPROVED** [review]
Thanks for the PR! Left a couple of comments on ReviewNB, let me know if you don't see them properly.

**Comment by sgugger** [comment]
Very nice!

**Comment by Rocketknight1** [comment]
<p>"In future" is allowable! (But only in British English) </p>

---

 <a href='https://app.reviewnb.com/huggingface/notebooks/pull/243/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by sgugger** [comment]
Good thing we use American English then :-p

**Comment by LysandreJik** [comment]
Looks great!

**Review by LysandreJik — APPROVED** [review]

**Comment by Rocketknight1** [comment]
Update: I incorporated @rmrao's feedback and also some suggestions from [Sergey Ovchinnikov](https://twitter.com/sokrypton). The whole thing should now run on a 16GB Colab GPU!

**Pull request merged**